### PR TITLE
hashtag timeline

### PR DESCRIPTION
**Pull request by yuskara**
new file:   src/components/HashtagTimeline/index.tsx
        modified:   src/components/index.ts
        new file:   src/data/twitter-data.json
        new file:   src/utils/getTwitterVideoUrl/index.ts
        modified:   src/utils/index.ts

### CODE REVIEW

File: src/components/HashtagTimeline/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                    {data.text}
+                  </Text>
+                  {videoVariants ? (
+                    <video controls muted width="250">
+                      <source
+                        src={getTwitterVideoUrl(videoVariants as any)}
+                        type={'video/mp4'}
+                      />
+                      Your browser does not support the video tag.
+                    </video>
```
Comment (7alip) [inline src/components/HashtagTimeline/index.tsx:55]: Can we use [React Player](https://github.com/cookpete/react-player) here?
You can take a look at [this example](https://codesandbox.io/s/sharp-pasteur-5n0jw). It looks very easy to add. And `light` attribute allows you to preview image of the video which is already provided in the twitter data. Please check the styling before committing.

```jsx
<ReactPlayer
  url="<twitter-video-url>"
  light="<twitter-image-url>"
/>
```

**Review by 7alip — CHANGES_REQUESTED** [review]
Thank you, it looks great! I have only one comment.

File: src/components/HashtagTimeline/index.tsx (hunk 1 above)
Comment (yuskara) [reply src/components/HashtagTimeline/index.tsx]: react-player has added

**Comment by 7alip** [comment]
Closing because of the branch name rules. Moved to => #67

**Pull request closed without merging**